lib/analysis/network.py

```diff
@@ -38,8 +37,24 @@ def message_number_graph(log_dict, nicks, nick_same_list):
     for day_content_all_channels in log_dict.values():
         for day_content in day_content_all_channels:
             day_log = day_content["log_data"]
+            today_conversation = [[0] for i in range(config.MAX_EXPECTED_DIFF_NICKS)]
             for line in day_log:
                 flag_comma = 0
+
+                def helper(nick_sender, nick_receiver, conversation):
```

Comment:
please name the function correctly. Place in correctly in util and documentation?